server/src/Commands.def

```diff
@@ -12,7 +12,9 @@
 #error Please define the 'X(id, command, description)' macro before inclusion!
 #endif
 
-X(Candidates, "candidates", "print completion candidates (require previous complete)")
+X(Candidates, "candidates",
+  "PREFIX STYLE - print completion candidates (require previous complete). "
+  "STYLE is \"\", \"insensitive\" or \"smart\"")
```

Comment:
This line is not correct. I think it's `exact`, `case-insentitive` and `smart-case`.